### PR TITLE
Enable multiple robots

### PR DESCRIPTION
**Pull request by YueErro**
Enable multiple robots
Add urdfs:
  - two_mara_robots
  - two_mara_robots_gripper_140_no_table

Adapt bridge

### CODE REVIEW

File: hros_cognition_mara_components/config/motors.yaml [hunk 1]
```diff
@@ -5,6 +5,12 @@ simulated_motors:
   - "/hrim_actuation_servomotor_000000000002/trajectory_axis2"
   - "/hrim_actuation_servomotor_000000000003/trajectory_axis1"
   - "/hrim_actuation_servomotor_000000000003/trajectory_axis2"
+  # - "/hrim_actuation_servomotor_000000000005/trajectory_axis2"
```
Comment (ahcorde) [inline hros_cognition_mara_components/config/motors.yaml]: Remove these lines and create a config file for two robots

File: hros_cognition_mara_components/config/motors.yaml [hunk 2]
```diff
@@ -13,3 +19,17 @@ real_motors:
...
   - "/hrim_actuation_servomotor_70B3D521A0A4/trajectory_axis2"
+
+motors:
+  - "motor1"
+  - "motor2"
+  - "motor3"
+  - "motor4"
+  - "motor5"
+  - "motor6"
+  # - "mara2_motor1"
```
Comment (ahcorde) [inline hros_cognition_mara_components/config/motors.yaml]: Remove these lines and create a config file for two robots

File: mara_description/urdf/reinforcement_learning/mara_robot_gripper_140_camera_train.urdf [hunk 3]
```diff
@@ -852,22 +852,15 @@
...
+    <plugin filename="librobotiq_gripper_gazebo_plugin.so" name="hrim_actuator_gripper_000000000004">
+      <kp>600</kp>
       <ki>0.1</ki>
       <kd>0.01</kd>
       <joint>robotiq_arg2f_base_to_left_inner_knuckle</joint>
       <joint>robotiq_arg2f_base_to_left_outer_knuckle</joint>
       <joint>robotiq_arg2f_base_to_right_inner_knuckle</joint>
       <joint>joint_finger</joint>
     </plugin>
-    <plugin filename="libgazebo_ros_joint_state_publisher.so" name="mara_joint_state">
```
Comment (ahcorde) [inline mara_description/urdf/reinforcement_learning/mara_robot_gripper_140_camera_train.urdf]: Why are you removing this plugin?

Comment (YueErro) [reply mara_description/urdf/reinforcement_learning/mara_robot_gripper_140_camera_train.urdf]: It is using a different plugin, [mara_gazebo_train.urdf.xacro](https://github.com/AcutronicRobotics/MARA/blob/master/mara_description/urdf/reinforcement_learning/mara_gazebo_train.urdf.xacro).
That [plugin](https://github.com/AcutronicRobotics/MARA/blob/master/mara_description/urdf/reinforcement_learning/mara_gazebo_run.urdf.xacro) is only used in `run` models.

Comment (rkojcev) [reply mara_description/urdf/reinforcement_learning/mara_robot_gripper_140_camera_train.urdf]: In the RL for training we are using different approach where we do not need this plugin.

**Pull request merged**